### ISSUE TITLE
Fallback Icon is higher priority than known spells?

### ISSUE THREAD

**Issue by Konungrr** [Sellorio/mega-macro#228]
I design my macros so that every possible combination of abilities and talents is covered and that every ability has it's own hotkey. I'm pretty sure the Fallback Icon function broke with 1.6.0, but as I was in the middle of M Fyrakk prog, I just rolled back to 1.5.11 and never bothered going back to figure out the fix until pre-patch forced the issue. 

My understanding of the built in icon/tooltip is that it follows the this priority:
1) A detailed out #showtooltip, such as '#showtooltip [mod:ctrl]Grand Expedition Yak;[mod:alt]Mighty Caravan Brutosaur;Flight Form'
2) An icon selected in the menu _without_ the 'Fallback Icon' box checked
3) Whatever spell or ability the macro would fire under those conditions
4) An icon selected in the menu _with_ the 'Fallback Icon' box checked.

Unfortunately, it seems that the priority list has shifted so that 'Fallback Icon' is overwriting the actual macro. For example, below is one of my utility macros. 

/use [mod:ctrl,known:Heart of the Wild]Heart of the Wild;[known:Renewal,noknown:Wild Charge][mod,known:Renewal]Renewal;[@mouseover,exists,known:Wild Charge][known:Wild Charge]Wild Charge

It has 'Wild Charge' as the baseline, with 'Renewal' on alt and 'Heart of the Wild' on Ctrl. If I don't have 'Wild Charge', it will default to 'Renewal', 'Heart of the Wild' stays on Ctrl. I have this macro assigned the 'Heart of the Wild' as 'Fallback Icon'. It should only show me 'Heart of the Wild' if I have not taken 'Wild Charge' or 'Renewal'. 

Currently, whenever I have the MegaMacro window open, it shows the correct icon. But when the window is closed, it reverts to 'Heart of the Wild' icon with 'Wild Charge' tooltip, but the icon switches to the 'Fallback Icon'. 

![WoWScrnShot_080924_183930](https://github.com/user-attachments/assets/096a576d-7b32-4be4-bbb4-253dfb1bf036)
![WoWScrnShot_080924_183938](https://github.com/user-attachments/assets/a8aa8316-0090-4aea-8f46-a6a4a33318b4)
![WoWScrnShot_080924_183942](https://github.com/user-attachments/assets/25bbd84e-1cac-4e1d-80a3-db55bc3d8f5d)

**Comment by bpapuni** [comment]
Have you updated to version 1.6.3? I just tried to recreate the issue but the correct icon displayed for me in each scenario you described. I'll take another look when I get home from work.

**Comment by Konungrr** [comment]
Yes, version 1.6.3. The correct icon displayed with the MegaMacro window closed?

**Comment by Khamulus** [comment]
Ive the same problem (latest version installed).

The icons are switching in fight, when entering fight of after login.
When i open the megamacro window, all icons return to the correct state ... but they change back to the questionmark sometimes.

Edit:
I noticed that this only happens on newer macros (since Pre-Patch) for me. The Macros on my Druid work perfectly (was Main in DF), but on my new Warlock, the macros show the descriped problem
